File: (X) Adicionales/(X.2) Práctica para laboratorios/(X.2.1) Ejercicios de Pascal/G1/1-1.pas
{* Escribir un programa en pascal que dado dos números enteros realice la
suma, resta, multiplicación, división entera y resto de los mismos. Los
números son: X = 55 e Y = 30. *}

program EJ1;

var
	x, y: integer;
	suma, resta, mult, div_ent, resto: integer;

begin
	x := 55;
	y := 30;

	suma := x + y;
	resta := x - y;
	mult := x * y;
	div_ent := x div y;
	resto := x mod y;

	writeln('Valor de X: ', x);
	writeln('Valor de Y: ', y);

	writeln();
	writeln('Suma: ', suma);
	writeln('Resta: ', resta);
	writeln('Multiplicación: ', mult);
	writeln('División entera: ', div_ent);
	writeln('Resto: ', resto);
end.

File: (X) Adicionales/(X.2) Práctica para laboratorios/(X.2.2) Ejercicios de C/G1/1-1.c
/* Escribir un programa en C que, dado dos números enteros, realice la suma,
resta, multiplicación, división entera y resto de los mismos. Los números
son: X = 132 e Y = 34. */

#include <stdio.h>

int main(void) {
    int x = 132;
    int y = 34;

    int suma = x + y;
    int resta = x - y;
    int mult = x * y;
    int div_ent = x / y;
    int resto = x % y;

    printf("Valor de X: %d\n", x);
    printf("Valor de Y: %d\n", y);

    printf("\n");
    printf("Resultado de la suma: %d\n", suma);
    printf("Resultado de la resta: %d\n", resta);
    printf("Resultado de la multiplicación: %d\n", mult);
    printf("Resultado de la división entera: %d\n", div_ent);
    printf("Resultado del resto: %d\n", resto);

    return 0;
}
